tests/snippets/ints.py

```diff
@@ -52,6 +52,11 @@
 # real/imag attributes
 assert (1).real == 1
 assert (1).imag == 0
+# numerator/denominator attributes
+assert (1).numerator == 1
+assert (1).denominator == 1
+assert (10).numerator == 10
+assert (1).denominator == 1
```

Comment:
did you meant to write `assert (10).denominator == 1` ?  
Overwise you just repeated from 2 lines above.

Reply:
Yes that was supposed to be a 10